docs/tactics.md

```diff
@@ -1053,3 +1053,29 @@ localized "attribute [simp] le_refl" in le
 ### rotate
 
 `rotate` moves the first goal to the back. `rotate n` will do this `n` times.
+
+### lift
```

Comment:
Could you illustrate this with some examples, printing the tactic state before and after calls? I get the idea, but it's a little hard to follow with `h t p type_p k hk` floating around.

Reply:
Yes, it was written very well. I rewrote it from scratch using only examples which get increasingly complicated.

Reply:
Thanks, this looks much nicer! You could copy the test here, and rewrite the guards as comments to show what's going on. I think it would be helpful but not completely necessary.

Reply:
Added some examples.